book/src/guides/textobject.md

```diff
@@ -25,6 +25,8 @@ The following [captures][tree-sitter-captures] are recognized:
 | `parameter.inside` |
 | `comment.inside`   |
 | `comment.around`   |
+| `element.inside`   |
+| `element.around`   |
```

Comment:
```suggestion
| `entry.inside`     |
| `entry.around`     |
```

I think this is a typo

Reply:
I had originally called it "element" but changed to "entry" after some thought. I just missed this spot when changing it, thanks for catching!